components/treeview/overview.md

```diff
@@ -91,7 +91,123 @@ The <a href="https://www.telerik.com/blazor-ui/treeview" target="_blank">Blazor
     }
 }
 ````
+>caption Expand nodes programmatically
 
+````CSHTML
+@*To expand a specific node, the parent of this node needs to be expanded as well.*@
+
+<p>
+    <label>Item Id: <TelerikNumericTextBox @bind-Value="@ItemToExpand" Width="80px" /></label>
+    <TelerikButton OnClick="@ExpandItem">Expand Item @ItemToExpand</TelerikButton>
+</p>
+<p>
+    <TelerikButton OnClick="@ExpandRoot">Expand Root Items</TelerikButton>
+    <TelerikButton OnClick="@ExpandAll">Expand All</TelerikButton>
+    <TelerikButton OnClick="@CollapseAll">Collapse All</TelerikButton>
+</p>
+
+<TelerikTreeView Data="@FlatData"
+                 @bind-ExpandedItems="@ExpandedItems"
+                 @bind-SelectedItems="@SelectedItems"
+                 SelectionMode="@TreeViewSelectionMode.Single" />
+
+@code {
+    public IEnumerable<TreeItem> FlatData { get; set; }
+    public IEnumerable<object> ExpandedItems { get; set; } = new List<TreeItem>();
+    public IEnumerable<object> SelectedItems { get; set; } = new List<TreeItem>();
+
+    int ItemToExpand { get; set; }
```

Comment:
```suggestion
    int? ItemToExpand { get; set; }
```

Use nullable int here, so there is no confusion if the user did not type anything in the NumericTextBox. Otherwise, by default the NumericTextBox will display 0, however there is no such item.